app/controllers/coronavirus_form/data_export_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require "csv"
+
+class CoronavirusForm::DataExportController < ApplicationController
+  def show
+    respond_to do |format|
+      format.html { render controller_path }
+      format.csv { render csv: produce_csv(usage_statistics(params[:start_date], params[:end_date])), filename: "data-export" }
```

Comment:
Nitpick, perhaps use a do/end block? e.g.
```
format.csv do
  render csv: produce_csv(usage_statistics(params[:start_date], params[:end_date])),
                      filename: "data-export"
end
```

Reply:
This looks better.  Will update.